### PR TITLE
Promotion config changes

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR contains two different sets of work relating to the promotion content type.

**First:**
Removed Promotional Image config and added it back fresh. There was a collision with the source field: media_field_image_1 with the event content type. This was causing the install to error out and prevented images from being added to promotions.

**Second:**
Began incorporating work for the new promotion theming and placement. This includes:
- Add Global Display boolean field
- Add Display Title boolean field
- Add view to filter promotions who are not displayed globally.
- Changed body text field to be plain text.

**Install Recommendations:**
Since none of the sites have any promotions to this point I think we should delete both the promotion content type and promotional image media item using the site admin before deploying this change. I was unable to smoothly update the source reference field on my local without doing a fresh install.

## Metadata
| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | no
| Risk level | High
| Relevant links | https://thinkoomph.jira.com/browse/RIG-67

### CODE REVIEW

File: ecms_base/features/custom/ecms_promotions/config/install/field.storage.node.field_promotion_global_display.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - node
+id: node.field_promotion_global_display
+field_name: field_promotion_global_display
+entity_type: node
+type: boolean
+settings: {  }
+module: core
+locked: false
+cardinality: 1
+translatable: true
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_promotions/config/install/field.storage.node.field_promotion_global_display.yml:14]: This should be translatable false.

File: ecms_base/features/custom/ecms_promotions/config/install/field.storage.node.field_promotion_display_title.yml [hunk 2]
```diff
@@ -3,13 +3,12 @@ status: true
...
+field_name: field_promotion_display_title
 entity_type: node
-type: text_with_summary
+type: boolean
 settings: {  }
-module: text
+module: core
 locked: false
 cardinality: 1
 translatable: true
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_promotions/config/install/field.storage.node.field_promotion_display_title.yml:14]: This should be translatable false

**Review by ben-hamelin — CHANGES_REQUESTED** [review]
Couple notes on the translation settings. Overall this looks good to me, we can discuss options for the deploy.

**Review by pfrilling — COMMENTED** [review]
I think it looks good once Ben's changes have been addressed.

**Pull request merged**